### PR TITLE
Rs 2023.03.15

### PR DESCRIPTION
**Pull request by rschlaikjer**
pancake

### CODE REVIEW

File: script/QA/QA_21.txt [hunk 1]
```diff
@@ -36,97 +37,106 @@ Roger that, teacher!
...
 }
-[sha:e2d65fcc963f228d45180c6a8dafc0494e929d3a]{
+[offset:43707]{
 -- Page 211, Offset 43707.
 -- え。
-Huh?
+Huh?%{n}
 }
 [sha:9a2b258cb18816e2206016a243207cfcfb69a371]{
 -- Page 211, Offset 43708.
```
Comment (Petrikow) [inline script/QA/QA_21.txt:142]: 43709

>I understand now... cats... speed....

43710
>Completely pointless Q&A corners...

43711
>Here it comes... the Luminary Cat Phenomenon!

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt [hunk 2]
```diff
@@ -1,148 +1,162 @@
...
 -- Page 204, Offset 40010.
 -- 振り落とされる前に目的の部位に到達できた……！
-I reach my objective before I could be thrown off...!
+I manged to reach my objective without being thrown off!
 }
 [sha:5a6c14f40e8742cc1a1796ca37e3020d4d8682e8]{
 -- Page 205, Offset 40011.
 -- 「ここなら……！」
-"If I do this...!"
+"If I cut her here―――!"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:142]: Hmm, I read this as more "If I cut her here [it'll work]".
> This spot for sure!
> This'll do it!

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt [hunk 3]
```diff
@@ -287,72 +307,82 @@ Since I've been found out, I need to run away as quick as possible―――!
...
+"Ah, of course.
 }
 [sha:481bec44499edc5ab481094276faa6af0e426fe6]{
 -- Page 218, Offset 40056.
 -- 目に見えるものでないと分からないんだっけ。
-You can't understand it if it's something you can't see.
+You can't understand what you can't see.
 }
 [sha:78738d9e60c51ff5a86b76d2962e809413475f59]{
 -- Page 218, Offset 40057.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:388]: Doesn't lead into the next page as smoothly imo

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:141]: The last one is fine.

File: script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_3_1.txt:388]: Just remove 'then'

**Pull request merged**